src/adminactions/merge.py

```diff
@@ -22,7 +22,7 @@
 from .utils import clone_instance
 
 
-class MergeForm(GenericActionForm):
+class MergeFormBase(forms.Form):
```

Comment:
This was changed to allow me to access the extra Merge Form fields without a model instance during validation